src/System.Private.ServiceModel/tests/Scenarios/Security/TransportSecurity/Https/HttpsTests.4.1.0.cs

```diff
@@ -172,7 +172,7 @@ public static void SameBinding_Soap12_EchoString()
                nameof(Client_Certificate_Installed),
                nameof(SSL_Available))]
     [Issue(959, Framework = FrameworkID.NetNative)] // Server certificate validation not supported in NET Native
-    [Issue(1295, OS = OSID.AnyUnix)] // A libcurl built with OpenSSL is required
+    [Issue(1295, OS = OSID.AnyCentOS | OSID.AnyFedora | OSID.AnyOpenSUSE | OSID.AnyOSX | OSID.AnyRHEL)] // A libcurl built with OpenSSL is required
```

Comment:
Just curious -- so all *nixes still fail except Ubuntu? So this fix is primarily to get Ubuntu working?